init.d/autossh

```diff
@@ -45,6 +45,13 @@ start() {
     eend $?
 }
 
+stop() {
```

Comment:
Absent a reason why the default stop function (which calls start-stop-daemon for us) is insufficient, I'm inclined to continue to let it use the default, which shows this when I stop it:

dmoulding@hitch ~ $ sudo rc-service autossh.gate.danm.net stop
 * Stopping autossh.gate.danm.net ...                                     [ ok ]